scripts/circle_helper.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function mapbox_time_start {
+    :
+}
+
+function mapbox_time_finish {
+    :
+}
+
+function mapbox_time {
+    :
+}
+
+function mapbox_fold {
+    :
+}
+
+function mapbox_nanoseconds {
+    :
+}
+
+export JOBS=${CIRCLE_NODE_TOTAL:-6}
```

Comment:
~~Yes - in that case we can just use export JOBS=${JOBS:-6}.~~
These are not needed at all in the end - JOBS is now exported from `circle.yml`.